src/cargo/core/shell.rs

```diff
@@ -423,6 +454,63 @@ impl ShellOut {
         Ok(())
     }
 
+    /// Prints out a message with a status to stdout. Output the specified color and bold for the query content.
+    fn message_stdout(
+        &mut self,
+        name_no_query: Vec<&str>,
+        desc_no_query: Vec<&str>,
+        query: &str,
+        color: Color,
+        justified: bool,
```

Comment:
This is always `true` right now so does this need to be a parameter?

Reply:
Because there are cases where color output is not supported. 
![image](https://user-images.githubusercontent.com/46313511/147206500-c6733edf-87d4-4f43-bb98-6d066c734bf4.png)

I learned the way of the ```message_stderr()```function.